src/Kunstmaan/FixturesBundle/Builder/MediaBuilder.php

```diff
@@ -18,7 +18,7 @@ class MediaBuilder implements BuilderInterface
     private $fileHandler;
 
     /**
-     * @var MimeTypeGuesserInterface
+     * @var MimeTypes
```

Comment:
We should again allow injecting both classes in the constructor and deprecate the old class injection there